src/libostree/ostree-mutable-tree.c

```diff
@@ -146,7 +149,7 @@ invalidate_contents_checksum (OstreeMutableTree *self)
       break;
 
     g_free (self->contents_checksum);
```

Comment:
You can drop this line too, that's the beauty of `g_clear_pointer` :)

Reply:
Fixed